### ISSUE TITLE
Feature request: Zoom out

### ISSUE THREAD

**Issue by dark-passages** [satishsampath/wink-issues#13]
After making full screen screencaptures the Wink window shows the captured screens, but the viewport is not big enough to show the entire captured images so there are scroll bars on the sides. It would be amazing if I could show the full picture inside the viewport (without the need for scrollbars).

**Comment by satishsampath** [comment]
I'm going to defer this for a future release, because it is a lot of work :) Keeping it open though.

**Comment by dark-passages** [comment]
Yeah, it just came to mind when I was testing the app, would never expect it to be in the next release. (like all other feature requests I added)
